Mathlib/Tactic/Positivity/Core.lean

```diff
@@ -236,8 +236,7 @@ def compareHypLT (lo e : Q($α)) (p₂ : Q($lo < $e)) : MetaM (Strictness zα p
   | _ => pure .none
 
 /-- A variation on `assumption` when the hypothesis is `a = b` where `a` is a numeral. -/
-def compareHypEq (e a b : Q($α)) (p₂ : Q($a = $b)) : MetaM (Strictness zα pα e) := do
-  let .defEq _ ← isDefEqQ e b | return .none
+def compareHypEq (e a : Q($α)) (p₂ : Q($a = $e)) : MetaM (Strictness zα pα e) := do
```

Comment:
Is `b` now garbage?

Reply:
I do the defeq check that was also deleted earlier for some matching or something of the like, so it's no longer needed here

Reply:
Can you remove it from the docstring?